client/app/scripts/liveblog-edit/directives.js

```diff
@@ -54,16 +55,24 @@ define([
                 DraftPostsController.$inject = ['$scope'];
                 function DraftPostsController($scope) {
                     var mv = this;
+                    mv.posts = [];
+                    mv.postsInfo = {};
                     mv.selectDraftPost = $scope.onDraftPostSeleted;
                     function updateList() {
-                        postsService.getDrafts($scope.blog._id).then(function (posts) {
-                            mv.posts = posts;
-                        });
+                        postsService.fetchDrafts(
+                            $scope.blog._id,
+                            mv.posts,
+                            mv.postsInfo
+                        );
                     }
+                    $scope.$on('posts', function() {
```

Comment:
I would use a more explicit name for these events.
What about `lb.posts.updated` ?